README.adoc

```diff
@@ -461,12 +516,63 @@ operator-sdk build $IMAGE --enable-tests && docker push $IMAGE
 operator-sdk test cluster $IMAGE --namespace $NAMESPACE --service-account unifiedpush-operator
 ....
 
+== Makefile command reference
+
+=== Application Commands
+
+|===
+| *Command*                        | *Description*
+| `make install`                   | Creates the `{namespace}` namespace, application CRDS, cluster role and service account.
+| `make uninstall`                 | Uninstalls the operator and the Service and DB. Deletes the `{namespace}`` namespace, application CRDS, cluster role and service account and the app namespace. i.e. all configuration applied by `make install`
```

Comment:
I think there's a extra backtick after `{namespace}`
```suggestion
| `make uninstall`                 | Uninstalls the operator and the Service and DB. Deletes the `{namespace}` namespace, application CRDS, cluster role and service account and the app namespace. i.e. all configuration applied by `make install`
```
Extra back tick here and in the other commands too

Reply:
👍